sources/HUBComponentWrapper.h

```diff
@@ -167,6 +167,9 @@ willUpdateSelectionState:(HUBComponentSelectionState)selectionState;
 /// Whether the wrapped component's view has appeared since the model was last changed
 @property (nonatomic, readonly) BOOL viewHasAppearedSinceLastModelChange;
 
+/// Whether the wrapped component's view has appeared for the first time
+@property (nonatomic, assign, nullable, readonly) NSNumber *viewHasAppearedForTheFirstTime;
```

Comment:
Because it can have 3 states:
* `nil` - `viewWillAppear` hasn't been called,
* `true` - `viewWillAppear` has been called only once,
* `false` - `viewWillAppear` has been called more than 1 time.

It's needed for example when a content operation is rescheduled. In the situation, when the component has appeared before and `contentOffset` hasn't been changed, we don't want to notify a `contentOffsetObserver`. We want to only notify it on the first appearance.